Sources/Views/ALKInformationCell.swift

```diff
@@ -76,31 +84,110 @@ final class ALKInformationCell: UITableViewCell {
+        setUpConstraintsForRating()
+        setupStyle()
     }
 
     fileprivate func setupConstraints() {
-        contentView.addViewsForAutolayout(views: [messageView, bubbleView])
+        contentView.addViewsForAutolayout(views: [messageView])
         contentView.bringSubviewToFront(messageView)
 
         messageView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 8).isActive = true
         messageView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -8).isActive = true
         messageView.centerXAnchor.constraint(equalTo: contentView.centerXAnchor).isActive = true
         messageView.widthAnchor.constraint(lessThanOrEqualToConstant: 300).isActive = true
-
-        bubbleView.topAnchor.constraint(equalTo: messageView.topAnchor, constant: 3).isActive = true
-        bubbleView.bottomAnchor.constraint(equalTo: messageView.bottomAnchor, constant: -3).isActive = true
-        bubbleView.leftAnchor.constraint(equalTo: messageView.leftAnchor, constant: -4).isActive = true
-        bubbleView.rightAnchor.constraint(equalTo: messageView.rightAnchor, constant: 4).isActive = true
+    }
+    
+    fileprivate func setUpConstraintsForRating() {
+        
+        let horizontalStackView = UIStackView()
+        let verticalStackView = UIStackView()
+        let lineViewLeft = UIView()
+        let lineViewRight = UIView()
+        
+        verticalStackView.axis  = NSLayoutConstraint.Axis.vertical
+        verticalStackView.distribution  = UIStackView.Distribution.equalSpacing
+        verticalStackView.alignment = UIStackView.Alignment.center
+        verticalStackView.spacing   = -5
+        
+        verticalStackView.addArrangedSubview(messageView)
+        if !commentTextView.text.isEmpty {
+            verticalStackView.addArrangedSubview(commentTextView)
+        }else{
+            verticalStackView.removeArrangedSubview(commentTextView)
+        }
+        
+        horizontalStackView.axis  = NSLayoutConstraint.Axis.horizontal
+        horizontalStackView.distribution  = UIStackView.Distribution.equalSpacing
+        horizontalStackView.alignment = UIStackView.Alignment.center
+        horizontalStackView.spacing   = 10
+
+        horizontalStackView.addArrangedSubview(lineViewLeft)
+        horizontalStackView.addArrangedSubview(verticalStackView)
+        horizontalStackView.addArrangedSubview(lineViewRight)
+        
+        lineViewLeft.backgroundColor = .lightGray
+        lineViewRight.backgroundColor = .lightGray
+        
+        contentView.addViewsForAutolayout(views: [horizontalStackView])
+        contentView.bringSubviewToFront(messageView)
+        
+        horizontalStackView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 10).isActive = true
+        horizontalStackView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -10).isActive = true
+        
+        lineViewLeft.widthAnchor.constraint(equalToConstant: 60).isActive = true
```

Comment:
Same here, move all the values to a separate enum.

Reply:
added enum for padding